src/main/capi/duckdb_value-c.cpp

```diff
@@ -151,6 +151,30 @@ duckdb_value duckdb_create_timestamp(duckdb_timestamp input) {
 duckdb_timestamp duckdb_get_timestamp(duckdb_value val) {
 	return {CAPIGetValue<duckdb::timestamp_t, LogicalTypeId::TIMESTAMP>(val).value};
 }
+duckdb_value duckdb_create_timestamp_tz(duckdb_timestamp input) {
+	return WrapValue(new duckdb::Value(duckdb::Value::TIMESTAMPTZ(duckdb::timestamp_t(input.micros))));
+}
+duckdb_timestamp duckdb_get_timestamp_tz(duckdb_value val) {
+	return {CAPIGetValue<duckdb::timestamp_t, LogicalTypeId::TIMESTAMP_TZ>(val).value};
+}
```

Comment:
All the new getters in this file need a `nullptr` check. And the respective test case.
```c
if (!val) {
	return the_function_return_type {0};
}
```
Later, in the `test/api/capi/test_capi_complex_types.cpp` tests.
```c
REQUIRE(duckdb_the_function_return_type(nullptr).field == 0);
```